src/components/AppContainer.js

```diff
@@ -0,0 +1,111 @@
+import React, { Component } from 'react';
+import _ from 'lodash';
```

Comment:
I wanted to emphasize using lodash'es function (e.g. isNaN(), not the original one). So should I use ```js import isNaN from 'lodash/isNaN'; ``` rather than this approach?

Reply:
the common practice is 
`import { isNan, isEmpty, otherMethod } from 'lodash'`